app/assets/stylesheets/sulFooter.scss

```diff
@@ -0,0 +1,71 @@
+$stanford-footer-height: 250px;
+$stanford-footer-md-height: 140px;
+$sul-footer-height: 80px;
+
+#su-content {
+  height: 100%;
+  padding-bottom: $stanford-footer-height + $sul-footer-height !important;
+
+  @media (min-width: 768px) {
+    padding-bottom: $stanford-footer-md-height + $sul-footer-height !important;
+  }
+}
+
+#sul-footer-container {
+  background-color: $white;
+  -webkit-box-shadow: 0 4px 8px -8px rgba(0, 0, 0, .2) inset;
+  box-shadow: 0 4px 8px -8px rgba(0, 0, 0, .2) inset;
+  clear: both;
+  margin-top: -1 * ($stanford-footer-height + $sul-footer-height);
+  min-height: $stanford-footer-height + $sul-footer-height;
+  margin-left: 50px;
+
+  @media (min-width: 768px) {
+    margin-top: -1 * ($stanford-footer-md-height + $sul-footer-height);
+    min-height: $stanford-footer-md-height + $sul-footer-height;
+  }
+  padding: 0 !important;
+  position: relative;
+  width: 100%;
+}
```

Comment:
An advantage of the newest sulFooter.scss is that it uses flexbox to position the footers instead of leaning on math that relies on knowing the footer heights.